_data/people.yml

```diff
@@ -291,4 +291,12 @@
     now: Timescale
     url: https://www.linkedin.com/in/matvey-arye-70a270b5
     picture: /assets/img/people/placeholder.png
+    
+   - first_name: Dongsheng
```

Comment:
@yangdsh I think you have an extra space on the "first_name" line, which is preventing the site from building. (YAML can be finicky.) If you fix that, I think it will build, and we can merge this in.